overdue/src/main/java/org/killbill/billing/overdue/wrapper/OverdueWrapper.java

```diff
@@ -100,6 +100,11 @@ public OverdueState refresh(final DateTime effectiveDate, final InternalCallCont
     }
 
     private OverdueState refreshWithLock(final DateTime effectiveDate, final InternalCallContext context) throws OverdueException, OverdueApiException {
+        if (overdueStateApplicator.isAccountTaggedWith_OVERDUE_ENFORCEMENT_OFF(context)) {
+            log.debug("OverdueStateApplicator: apply returns because account (recordId={}) is set with OVERDUE_ENFORCEMENT_OFF", context.getAccountRecordId());
+            return null;
```

Comment:
Shouldn't a clear state be returned instead?
That being said, it doesn't look like the return value is used. You could also just change this to return `void`.

Reply:
Fixed in https://github.com/killbill/killbill/pull/1706/commits/4b7051cdf469c63a5fefcc7eaf19e0a632850fc4